Doc/library/stdtypes.rst

```diff
@@ -3616,6 +3616,10 @@ copying.
       supports all format strings, including those that are not in
       :mod:`struct` module syntax.
 
+      *Order* can be {'C', 'F', 'A'}.  When *order* is 'C' or 'F', the data
+      of the original array is converted to C or Fortran order. 'A' returns
+      the data in the existing order. The default order is 'C'.
```

Comment:
Rather than "existing order", I would say "physical order" (or more precisely, "'F' if the buffer is Fortran-contiguous, 'C' otherwise"?).

Reply:
Yes, let's use the more precise version. 'A' *does* convert non-contiguous buffers to 'C', so it should be mentioned.